picky-asn1-x509/src/private_key_info.rs

```diff
@@ -19,32 +19,39 @@ use std::fmt;
 pub type EncapsulatedEcSecret = OctetStringAsn1;
 
 /// [Public-Key Cryptography Standards (PKCS) #8](https://tools.ietf.org/html/rfc5208#section-5)
+/// [Asymmetric Key Packages](https://tools.ietf.org/html/rfc5958#section-2)
```

Comment:
ED*/X* keys do require updated PrivateKeyInfo structure which now named OneSymetricKey. I decided to keep old naming but clarify things in comment - that should keep change more backwards-compatible

Reply:
If it makes sense, feel free to rename the actual struct to reflect that since we’ll release a new "breaking" version. However, you could add a type alias like this one instead:
```rust
#[deprecated(since = "0.10.0", note = "PrivateKeyInfo has been renamed into OneAsymmetricKey in order to reflect RFC accurately")]
pub type PrivateKeyInfo = OneAsymmetricKey;
```
This is still technically a breaking change since type aliases cannot be used in all places where a type is expected, but this will provide good-enough guidance to downstream user.

Reply:
Added alias described above, but feel free to request changes if you don't agree with the solution